dotcom-rendering/src/web/components/DynamicFast.stories.tsx

```diff
@@ -22,17 +22,246 @@ export default {
 	},
 };
 
-export const Default = () => (
+export const DF2VeryBigs2Bigs5Standards = () => (
 	<ContainerLayout
 		title="DynamicFast"
 		showTopBorder={true}
 		sideBorders={true}
 		padContent={false}
 		centralBorder="partial"
 	>
-		<DynamicFast trails={trails} showAge={true} />
+		<DynamicFast
+			groupedTrails={{
+				snap: [],
+				huge: [],
+				veryBig: [trails[0], trails[1]],
+				big: [trails[2], trails[3]],
+				standard: [
+					trails[4],
+					trails[5],
+					trails[6],
+					trails[7],
+					trails[8],
+				],
+			}}
+			showAge={true}
+		/>
 	</ContainerLayout>
 );
-Default.story = {
-	name: 'DynamicFast',
+
+DF2VeryBigs2Bigs5Standards.story = {
+	name: '2 Very Bigs - 2 Bigs - 5 Standards (None boosted)',
+};
+
+export const DF1Huge1VeryBig2Big3Standard = () => (
+	<ContainerLayout
+		title="DynamicFast"
+		showTopBorder={true}
+		sideBorders={true}
+		padContent={false}
+		centralBorder="partial"
+	>
+		<DynamicFast
+			groupedTrails={{
+				snap: [],
+				huge: [trails[0]],
+				veryBig: [trails[1]],
+				big: [trails[2], trails[3]],
+				standard: [trails[4], trails[5], trails[6]],
+			}}
```

Comment:
There is something to say for this api. It makes it clearer at this point what we're asking the component to do

Reply:
Yes! This is what eventually led me to decide to leave the `groupTrails` functionality in the enhancer rather than in the component itself - it adds a bit more complexity to our collection types but this interface is so readable from a testing perspective